optuna/importance/_mean_abs_shap.py

```diff
@@ -0,0 +1,79 @@
+from collections import OrderedDict
+from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Optional
+
+import pandas as pd
```

Comment:
pandas is an optional package in Optuna, could you move this import inside `try_import` block like this?
https://github.com/optuna/optuna/blob/c622be313074c0037fe3128ff5d20f9262c7af76/optuna/study/_dataframe.py#L14-L16

Reply:
By the way, can we implement this feature without pandas? Ideally, we would not like to use an optional dependency for usability to make installation easier..

Reply:
@nzw0301 Yep. I thought about that too. The usage of `pandas` is not mandatory, it is just that I'm more familiar with the module. I agree that it is better to keep the minimal dependency.